tests/commands/test_new.py

```diff
@@ -437,9 +440,10 @@ def test_tsv_with_flags_template_set_edit(repo: Repo) -> None:
     assert table_path.is_file()
 
     # create assets with table
-    ret = subprocess.run(['onyo', 'new', '--yes', '--edit', set_values,
-                          '--tsv', table_path, '--template', template],
-                         capture_output=True, text=True)
+    ret = subprocess.run(['onyo', 'new', '--yes', '--edit',
+                          '--keys', set_values, '--tsv', table_path,
+                          '--template', template], capture_output=True,
```

Comment:
Nit: when line-splitting, I prefer that it take into account context. It would make sense if the final line was
```
capture_output=True, text=True)
```